share/lib/python/neuron/__init__.py

```diff
@@ -164,27 +164,6 @@
     import sysconfig
     hoc_path = origin.rstrip("__init__.py") + "hoc" + sysconfig.get_config_var('SO')
     setattr(hoc, "__file__", hoc_path)
-  else:
-    # if the above is robust, maybe all this can be removed.
-    # next try is to derive from nrnversion(6) (only works for autotools build)
-    import platform
-    import os
-    p = h.nrnversion(6)
```

Comment:
So, this is not required because of autotools removal?

Reply:
exactly